index.html

```diff
@@ -1876,108 +1806,101 @@ <h2>Verification Methods</h2>
+of verification material properties are <code><a>publicKeyJwk</a></code> or
+<code><a>publicKeyBase58</a></code>. A verification suite definition is
+responsible for specifying the <a>verification method</a> <code>type</code> and
+its associated verification material. For example, see <a
+href="https://w3c-ccg.github.io/lds-jws2020/">JSON Web Signature 2020</a> and <a
+href="https://w3c-ccg.github.io/lds-ed25519-2018/">Ed25519 Signature 2018</a>.
+For all registered <a>verification method</a> types and associated  verification
+material available for <a>DIDs</a>, please see the DID Specification Registries
+[[?DID-SPEC-REGISTRIES]].
         </p>
 
         <p>
-This specification strives to limit the number of formats for expressing public
-key material in a <a>DID document</a> to the fewest possible, to increase the
-likelihood of interoperability. The fewer formats that implementers have to
+In order to increase the likelihood of interoperability, this specification
+limits the number of formats for expressing verification material in a <a>DID
+document</a> to the fewest possible. The fewer formats that implementers have to
 implement, the more likely it will be that they will support all of them. This
 approach attempts to strike a delicate balance between ease of implementation
-and supporting formats that have historically had broad deployment.
+and supporting formats that have historically had broad deployment. The two most
+common supported verification material properties are listed below:
         </p>
 
-        <p>
-A suite definition is responsible for specifying the <a>verification method</a>
-<code>type</code> and proof <code>type</code>. For example, see
-<a href="https://w3c-ccg.github.io/lds-jws2020/">JSON Web Signature 2020</a> and
-<a href="https://w3c-ccg.github.io/lds-ed25519-2018/">Ed25519 Signature
-2018</a>. For all registered and supported <a>verification method</a> types
-available for <a>DIDs</a>, please see the DID Specification Registries
-[[?DID-SPEC-REGISTRIES]].
-        </p>
+        <dl>
+          <dt><dfn>publicKeyBase58</dfn></dt>
+          <dd>
+            <p>
+The <code>publicKeyBase58</code> property is OPTIONAL. This feature is
+non-normative. If present, the value MUST be a <a
+data-cite="INFRA#string">string</a> representation of a [[?BASE58]] encoded
+public key.
+            </p>
+            <p class="issue atrisk"
+               title="publicKeyBase58 and publicKeyMultibase">
+The DID Working Group is seeking implementer feedback on the preference of the
+ecosystem with respect to using <code>publicKeyBase58</code> [[?BASE58]]  or
+<code>publicKeyMultibase</code> [[?MULTIBASE]]. The latter can be used for
+encoding more base-representation formats and provides a more future proof path.
+Depending on implementer feedback, one or both options might be included in the
+final specification, or migrated into the DID Specification Registries as an
```

Comment:
Might it be useful to note that this decision ultimately falls upon defacto consensus established via the creation of new LDP suites rather than referencing the DID Spec Registries as the go to place for it? Even though the LDP work has significant work to be done still, I think its useful to imply the connection between the two standards now.

Reply:
Probably worth noting that publicKeyBase58 is not going to be removed from older suites, but instead that new suites will only support publicKeyMultibase instead.

Reply:
Well, yes to all the above -- but that's the sort of feedback we're asking for. We may not want to presume that we're going to get that sort of feedback. Would you mind taking a shot at rewording this in another PR, @kdenhartog ?